doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_srv.rst

```diff
@@ -305,28 +305,12 @@ This allows the Light Lightness Server to adjust its output level that is based
 
    Light Lightness Control Server illuminance regulator
```

Comment:
If we are not keeping the illuminance regulator name, we should probably align in this file.